samples/excel/85-preview-apis/data-types-references.yaml

```diff
@@ -0,0 +1,339 @@
+          {
+            employeeID: 1,
+            lastName: "Davolio",
+            firstName: "Nancy",
+            title: "Vice President, Sales",
+            reportsTo: null
+          },
+          {
+            employeeID: 2,
+            lastName: "Fuller",
+            firstName: "Andrew",
+            title: "Sales Representative",
+            reportsTo: 1
+          },
+          {
+            employeeID: 3,
+            lastName: "Leverling",
+            firstName: "Janet",
+            title: "Sales Representative",
+            reportsTo: 4
+          },
+          {
+            employeeID: 4,
+            lastName: "Peacock",
+            firstName: "Margaret",
+            title: "Sales Manager",
+            reportsTo: 1
+          },
+          {
+            employeeID: 5,
+            lastName: "Buchanan",
+            firstName: "Steven",
+            title: "Sales Representative",
+            reportsTo: 4
+          },
+          {
+            employeeID: 6,
+            lastName: "Suyama",
+            firstName: "Michael",
+            title: "Sales Representative",
+            reportsTo: 4
+          },
+          {
+            employeeID: 7,
+            lastName: "King",
+            firstName: "Robert",
+            title: "Sales Representative",
+            reportsTo: 4
+          },
+          {
+            employeeID: 8,
+            lastName: "Callahan",
+            firstName: "Laura",
+            title: "Inside Sales Coordinator",
+            reportsTo: 1
+          },
+          {
+            employeeID: 9,
+            lastName: "Dodsworth",
+            firstName: "Anne",
+            title: "Sales Representative",
+            reportsTo: 4
+          }
+        ];
+    language: typescript
+template:
+    content: |
+        <section class="ms-font-m">
+          <p>This sample shows how to create entity values with references to other entity values. An entity value is a container for data, and this container can have references (or links) to other entity values on the same worksheet.</p>
+        </section>
+        <section class="setup ms-font-m">
+          <h3>Set up</h3>
+          <button id="setup" class="ms-Button">
+            <span class="ms-Button-label">Create table</span>
+          </button>
+        </section>
+        <section class="samples ms-font-m">
+          <h3>Try it out</h3>
+          <button id="add-entities-to-table" class="ms-Button">
+            <span class="ms-Button-label">Add entity values</span>
+          </button>
+          <p>To see the entity value, click the icon to the left of the title in the <strong>Employee</strong> column after
+            selecting <strong>Add entity values</strong>.</p>
+          <p>Inside the card modal window, select the <strong>Extract to grid</strong> icon to the right of the <strong>Manager</strong> field or the <strong>Direct Reports</strong> field to the grid to display the referenced entities.</p>
```

Comment:
Consider changing "display the referenced entities" to "add the referenced entities to the table" or something. "Display" makes me think it'll happen in the control or the task pane.